singlevenue/src/main/res/layout/activity_main.xml

```diff
@@ -106,6 +107,19 @@
                         android:gravity="center"
                         android:textAppearance="?android:attr/textAppearanceMedium" />
 
+                    <TextView
+                        android:id="@+id/level_nav_textLabel"
```

Comment:
This view looks very off-centred. Also I think that the text should state the short name of the level you are on to be more descriptive.